### ISSUE TITLE
Show failed query count in admin ui

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#2223]
Background: https://airtable.com/tblD3oZPLJgGhCmch/viwINAgXA5on9pr2T/recBvBMNIXc0mTxNn

**Comment by Amruta-Ranade** [comment]
@piyush-singh We have #26731 for this issue, but I am not sure if it's user-facing (yet - until we make it into a graph). Am I correct in assuming this issue doesn't have a doc impact at least for 2.1?

**Comment by piyush-singh** [comment]
That is correct. The only way to access this for now is via a custom chart in debug pages, so for now I don't think this will impact documentation.

**Comment by jseldess** [comment]
If we think this is a question that users commonly have, as a stop-gap for 2.1, we could add a [SQL FAQ](http://www.cockroachlabs.com/docs/dev/sql-faqs.html): `Can I get insight into the number of failed SQL queries?`, with the answer being to create a [customer debug chart](https://www.cockroachlabs.com/docs/v2.1/admin-ui-custom-chart-debug-page.html). Thoughts, @piyush-singh, @Amruta-Ranade?

**Comment by Amruta-Ranade** [comment]
SGTM!

**Comment by piyush-singh** [comment]
Makes sense on my end as well!

**Comment by couchand** [comment]
As @Amruta-Ranade points out, cockroachdb/cockroach#30371 changes the previous discussion.  There is now a chart on the SQL dashboard.
